tasks/db_deploy/orca_sql.py

```diff
@@ -396,8 +396,8 @@ def recovery_file_table_sql() -> TextClause:
             IS 'Name of the file being restored.';
         COMMENT ON COLUMN recovery_file.key_path
             IS 'Full key value of the data being restored.';
-        COMMENT ON COLUMN recovery_file.multipart_chunksize
-            IS 'Overrides orca_default_multipart_chunksize in TF.';
+        COMMENT ON COLUMN recovery_file.multipart_chunksize_mb
+            IS 'Overrides orca_default_multipart_chunksize_mb in TF.';
```

Comment:
Is this `orca_default_multipart_chunksize_mb` or `default_multipart_chunksize_mb` in terraform?

Reply:
Whoops, hadn't saved that properly. Fixed.